### DISCUSSION TITLE
Change minor layout of dunst notification

### DISCUSSION THREAD

**Discussion by mahaveergurjar** [prasanthrangan/hyprdots#1100] [Ideas]
I think this layout is see much better 
Can i change this like this layout for dunst

![240317_03h13m10s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/04efba4f-4fad-427c-b2d9-7b8b8e86a2db)

New layout 

![240317_03h07m48s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/949c3f76-e242-42e7-99a2-b5a10bc8e62e)

@prasanthrangan

**Comment by RookiexCookie** [comment]
Looks quite nice
you can make a PR about it tho

**Reply by mahaveergurjar** [reply]
PR Done
